src/components/Button/Button.tsx

```diff
@@ -149,13 +156,15 @@ export default class Button extends React.Component<ButtonProps, {}> {
     const href = (this.props as LinkButtonProps).href;
     const status = (this.props as RegularButtonProps).status;
     const isDisabled = status === ButtonStatus.DISABLED || status === ButtonStatus.LOADING;
+    const type = this.props.type || ButtonType.BUTTON;
 
     return (
       <BaseButton
         ariaLabel={ariaLabel}
         className={this.getClasses()}
         disabled={isDisabled}
         href={href}
+        type={href ? '' : type}
```

Comment:
We may not want to allow `type` when the `Button` is a link. @swese44 should we deal with this attribute in `RegularButtonProps` instead?

Reply:
I've now done `type?: undefined;` on `LinkButtonProps`.  Does that work?

Reply:
I think we've done `type?: void` for other attributes in `Button`.